packages/suite/src/components/suite/Metadata/index.tsx

```diff
@@ -15,7 +15,8 @@ type Props = {
 const Metadata = ({
     title = 'Trezor Suite',
     description,
-    image = `${SUITE_URL}${resolveStaticPath('images/meta.png')}`,
```

Comment:
here I am wondering why we use absolute path using SUITE_URL. It works in production, but it does not work on dev server (sldev). Is there any reason for this?